programs/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 set(programs_target "${MBEDTLS_TARGET_PREFIX}programs")
 add_custom_target(${programs_target})
 
+set(ssl_opt_target "${MBEDTLS_TARGET_PREFIX}ssl-opt")
```

Comment:
Minor/unsure: should we call this ssl-opt-prereq to make it clear that this target is not running `ssl-opt.sh` but just building it prerequisites?

Reply:
I like having a name that's easy to type, and `ssl-opt` is the name I use in `mbedtls-prepare-build`.